test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
Hmm how can both the old namespace and new namespace work? Does this project compile?